Documentation~/whats-new.md

```diff
@@ -1,4 +1,23 @@
-# What's new in version 5.0
+# What's New in ProBuilder
+
+For a full list of changes and updates in this version, refer to the [ProBuilder package changelog](https://docs.unity3d.com/Packages/com.unity.probuilder@latest/index.html?subfolder=/changelog/CHANGELOG.html).
+
+
+## What's new in version 6.0
+
+Summary of changes in ProBuilder package version 6.0:
+
+* Moved ProBuilder actions that apply to the entire ProBuilder mesh component, such as Export, Subdivide, and Center Pivot, to the [Scene view context menu](xref:SceneViewContextMenu). 
+* Moved ProBuilder actions that rely on element selection, such as Bridge, Weld Vertices, and Grow Selection, to the [Scene view context menu](xref:SceneViewContextMenu) when the ProBuilder tool context is active. Enable the ProBuilder tool context in the Tool overlay.  
+* Moved the Vertex, Edge, and Face edit modes to the Tool Settings overlay in the Scene view. To display ProBuilder edit modes in the Tool Settings overlay, enable the ProBuilder tool context in the Tools overlay.  
+* Moved creation and editing tools to the Tools overlay. 
+* Added a preview option for most ProBuilder actions.
+* Moved these tool options from the ProBuilder toolbar to the Tool Settings overlay:
+	* Select Hidden
+	* Orientation: Normal
+	* Rect: intersect
```

Comment:
I feel like this could be reordered in terms of importance. The elevator pitch of PB6 is basically that PB6 now has a dedicated tool context which removes the need for a PB window and moves functionality access points into the Tools overlay and SV contextual menus.

What I'd suggest is making the first points, somewhere long the lines of:
* PB6 now has a dedicated tool context which can be activated in the Tools overlay.
* PB6 tool context together with new scene view context menus removes the need for previous dedicated PB window.

then keep the current points but slighly reorder:

* Moved the Vertex, Edge, and Face edit modes to the Tool Settings overlay in the Scene view. To display ProBuilder edit modes in the Tool Settings overlay, enable the ProBuilder tool context in the Tools overlay.
* Moved ProBuilder actions that rely on element selection, such as Bridge, Weld Vertices, and Grow Selection, to the [Scene view context menu](xref:SceneViewContextMenu) when the ProBuilder tool context is active. Enable the ProBuilder tool context in the Tool overlay.
* Moved ProBuilder actions that apply to the entire ProBuilder mesh component, such as Export, Subdivide, and Center Pivot, to the [Scene view context menu](xref:SceneViewContextMenu).
* Moved creation and editing tools to the Tools overlay. 

and the rest are great I think.

One thing worth mention is that PolyShape creation tool has got some additional love, so adding something like "PolyShape workflow improvements" wouldn't hurt.

Reply:
This is fantastic, Mod. 

For PolyShape, any specific improvements or changes we can highlight?

Reply:
Creation tools improvements:
- With 'Create Cube/Sphere/...' tool you have now hold SHIFT after your last create shape to start your new shape as a copy of the previous.
- With 'Create PolyShape' you can now continue creating new shapes without needing re-enter the tool.